### ISSUE TITLE
Jiqiong Tang

### ISSUE THREAD

**Issue by aflies1** [promazo/Sourcing#179]

**Comment by aflies1** [comment]
Career?
All study and experience is around data analyst
Wants to graduate and work in that roles
Go from entry level to senior
Will apply for Bay Area / anywher
Major is finance, so something related would be grea

Timing?
Can join as soon as possible (CPT) 
2-4 months would not be aproblem

Python?
-Majoring in finance, loved working with data and finding insights
- Works with SQL and Tablue to analyze data
- Did a ML project predicting hpousing proces
Has alos exctracted data from finacial reports
Has used pythoin to extract data in some fo her work

SQL, R, SAS, and tableau

ML?
Has mainly been coursework
3-month project to complete
Worked with team members to create neural network and regression models to make predictions

Large Data sets?
Ml project has worked with more that 10K record of housing features and price
Personal Credit default prediction - worked with 80K personal record
Has experience with finiding missing values and converting data for further analysis

Schedule?
2-3 hours per day of availability?
Only 1 project, andjust needs to collaborate with her team members 
Morning or afternoon works well
Will need to meet with her professor 1 per week

Team? 
Many experiences working with classmates, her program focuses on group work, comfortable to schedule time with others, create a timeline to complete final results
Within her other internships she has collaborated as well.

**Comment by aflies1** [comment]
Resume; https://drive.google.com/drive/folders/1nWwkVJCtvCkvgcx3xfonrgqEFwSh67dO

**Comment by ldaudet** [comment]
Jiqiong Tang 

When she was in her undergraduate program, she took a lot of CS elective classes. She did so becasue she was very intersted in that. With that basic knowledge, she decided to add stem to her graduate degree.

Fairly strong accent, but strangely not bothering comprehension at all, at least to me. She did struggle at time to express complex stuff, looking for words sort of thing. 

She took some sQL and machine learning courses while in her masters. She enjoys working with data. She has experience with data visualisation in Python.

She used ML to analyse the text from a company financial reports. She shared her github with me https://github.com/tangjiqiong/ to show the work she s done in Python. It is a group project, not just her, and it is very succint, and not cleaned up, which is worrysome because she uses it as a crown jewl for her expertise.

I also checked out her other repo on the machine learning prediction of house prices, and her again it is very basic entro to ML sort of project like you see on Medium or Kaggle.

TL;DR: she has some python and coding experience, but very superficial for what she showed me. She could do on prescient in a pinch, since the project is not very complex, more on the tedious side, but with the quality of the other candidates, I think we would have better options.
